tests/aws/services/s3/test_s3_api.py

```diff
@@ -623,6 +624,47 @@ def test_upload_part_copy_range(self, aws_client, s3_bucket, snapshot):
                 )
             snapshot.match(f"upload-part-copy-range-exc-{src_range}", e.value.response)
 
+    @markers.aws.validated()
+    @pytest.mark.xfail(
+        condition=config.LEGACY_V2_S3_PROVIDER,
+        reason="Moto does not handle the exceptions properly",
+    )
+    @markers.snapshot.skip_snapshot_verify(paths=["$..PartNumberMarker"])  # TODO: invetigate this
```

Comment:
the test will fail, because the `DisplayName` field is not always returned, and depends on the region. However, I think LocalStack always returns it, and your snapshot was generated against a region which doesn't. We can ignore it. 
```suggestion
    @markers.snapshot.skip_snapshot_verify(
        # Not always present depending on the region
        paths=["$..Owner.DisplayName"],
    )
```

Reply:
yeah saw the warning here ... glad you understood why